vertical-pod-autoscaler/pkg/recommender/logic/recommender.go

```diff
@@ -147,3 +149,31 @@ func CreatePodResourceRecommender() PodResourceRecommender {
 		lowerBoundEstimator,
 		upperBoundEstimator}
 }
+
+// MapToListOfRecommendedContainerResources converts the map of RecommendedContainerResources into a stable sorted list
+// This can be used to get a stable sequence while ranging on the data
+func MapToListOfRecommendedContainerResources(resources RecommendedPodResources) *vpa_types.RecommendedPodResources {
```

Comment:
I initially placed it here because it is taking parameter of that same package.
It is a conversion function between a type of `logic` and a type of our api `v1` aka `vpa_types`, so placing it in yet another package `routines` looks strange to me. Later as a developper, if I am looking for a conversion function, this is here in `logic` I would search in a first place.
To be consistent, what I would propose is to move the test function to the `logic` package. In fact the test was initially written in the `routines` package because that function was test tested through the calling function `getCappedRecommendation` and the test is really only about this conversion function.
WDYT?